app/actions/StakePoolActions.js

```diff
@@ -1,13 +1,29 @@
 // @flow
 import Promise from "promise";
 import {
-  getPurchaseInfo, setStakePoolAddress, setVoteChoices, getNextAddress, getStakePoolInfo
+  getPurchaseInfo, setStakePoolAddress, setVoteChoices, getNextAddress, getStakePoolInfo, getAllStakePoolInfo,
 } from "wallet";
 import { getWalletCfg, updateStakePoolConfig } from "../config";
 import { importScriptAttempt } from "./ControlActions";
 import * as sel from "../selectors";
 import * as wallet from "wallet";
 
+export const GETSTAKEPOOLINFO_ATTEMPT = "GETSTAKEPOOLINFO_ATTEMPT";
+export const GETSTAKEPOOLINFO_FAILED = "GETSTAKEPOOLINFO_FAILED";
+export const GETSTAKEPOOLINFO_SUCCESS = "GETSTAKEPOOLINFO_SUCCESS";
```

Comment:
I thin for consistency, these should be named `GETSTAKEPOOLSTATS_XXX`, so we don't confuse the stakpoolStats vs stakepoolInfo branches of execution